entity-framework/ef6/modeling/code-first/data-annotations.md

```diff
@@ -82,11 +82,11 @@ Entity Framework supports composite keys - primary keys that consist of more tha
     }
 ```
 
-If you were to try and use the above class in your EF model you wuld get an InvalidOperationExceptions stating;
+If you were to try to use the above class in your EF model, you would get an `InvalidOperationException`:
```

Comment:
"If you were to try to use" sounds a bit weird still. I agree that what was there before wasn't better. What if we just simplify it to:

> If you try to use the above class in your EF model, you will get an `InvalidOperationException`:

Reply:
@divega Your proposal sounds much better.

Reply:
I agree, it still sounds weird. I left it because I had only intended on fixing the spelling error. 
Now that more than minimal time has been spent on this update, I guess I should just take the time to fix the rest of the oddities in the article. I'll send an update tomorrow.

Reply:
Thanks for taking the time to do this!

Reply:
Thanks for taking the time to do this!